### PR TITLE
Added ability to set the background-color of the spinner backdrop.

### PR DESCRIPTION
**Pull request by krishnakumarp**
Implements the ability to set the background-color of the spinner backdrop. Refer #143

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,7 @@ You can customize the following parameters:
   - The **extra duration** (ie. how many extra milliseconds should the spinner be visible).
   - The **minimum duration** (ie. how many milliseconds should the spinner be visible at least).
   - The spinner **opacity**.
+  - The spinner **background-color** (ie. the color of the spinner backdrop).
```
Comment (mpalourdio) [inline README.md:94]: That's the **backdrop** background color here, not the *spinner*

**Comment by mpalourdio** [comment]
Thanks for your contribution, I have just tested and there is a problem now :) When the backdrop is disabled (`[backdrop]="false"`), there's now a default background color behind the spinner. Will think about it a blit later.

File: README.md (hunk 1 above)
Comment (mpalourdio) [reply README.md]: I have fixed the doc

**Comment by mpalourdio** [comment]
I have amended your PR with some commits. Merging now, will release later! thanks a lot for contributing!

**Comment by krishnakumarp** [comment]
Thanks!

**Pull request merged**